apps/client/src/pages/home/Home.page.tsx

```diff
@@ -0,0 +1,37 @@
+import { ReactElement } from 'react';
+import { useNavigate, useParams } from 'react-router';
+
+export function Home(): ReactElement {
+  const { id } = useParams();
+
+  const navigate = useNavigate();
+
+  const getTournamentId = (): string => {
+    return '123';
+  };
+
+  const routeToRoom = (): void => {
+    if (id) {
+      navigate(`/${id}/room`);
+    } else {
+      const tournamentId = getTournamentId();
+      navigate(`/${tournamentId}/room`);
+    }
+  };
+
+  return (
+    <div>
+      <h1>Home</h1>
+      {id && <p>(id: {id})</p>}
```

Comment:
lets use a ternary operation with a null value instead of `&&`

Reply:
changed a0b11062df50938955e49355176719383d577e6a